proto/api/v2/compliance_profile_service.proto

```diff
@@ -54,10 +78,17 @@ service ComplianceProfileService {
         };
     }
 
-    // ListComplianceProfiles returns profiles matching given query
-    rpc ListComplianceProfiles (RawQuery) returns (ListComplianceProfilesResponse) {
+    // ListComplianceProfiles returns profiles matching given request
+    rpc ListComplianceProfiles (ProfilesForClusterRequest) returns (ListComplianceProfilesResponse) {
+        option (google.api.http) = {
+            get: "/v2/compliance/cluster/profiles"
+        };
+    }
+
+    // ListProfileSummaries returns profiles matching each cluster and the given query
```

Comment:
What query does it take other than cluster id?

Reply:
none.  No paging for now as the number of records is low